extensions/amp-facebook-comments/0.1/amp-facebook-comments.js

```diff
@@ -69,11 +75,36 @@ class AmpFacebookComments extends AMP.BaseElement {
     listenFor(iframe, 'embed-size', data => {
       this./*OK*/changeHeight(data['height']);
     }, /* opt_is3P */true);
+    this.unlistenMessage_ = listen(
+        this.win,
+        'message',
+        this.handleFacebookMessages_.bind(this)
+    );
+    this.toggleLoading(true);
     this.element.appendChild(iframe);
     this.iframe_ = iframe;
     return this.loadPromise(iframe);
   }
 
+  /** @private */
+  handleFacebookMessages_(event) {
+    if (this.iframe_ && event.source != this.iframe_.contentWindow) {
+      return;
+    }
+    let eventData = getData(event);
+    if (!eventData || !(isObject(eventData) || tryParseJson(eventData))) {
```

Comment:
I'm still having a little trouble quickly reading these `if` statements. Would something like this be equivalent?

```js
const eventData = getData(event);
if (!eventData) {
  return;
}

const parsedEventData = isObject(eventData) ? eventData : tryParseJson(eventData);
if (!parsedEventData) {
  return;
}
```

Reply:
Made the changes you asked for. Thanks for the suggestion!